ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -306,6 +306,9 @@ public void close() {
         shutDownExecutorService(scheduler, config.updaterShutDownTimeoutMs, TimeUnit.MILLISECONDS);
       }
       helixStore.stop();
+      if (notifier != null) {
```

Comment:
does calling stop on helixStore twice have any issues ?
the stop at L308 may be removed since we are closing it in the notifier

Reply:
notifier has a different helixstore.